### DISCUSSION TITLE
Feature dev branch request

### DISCUSSION THREAD

**Discussion by DeltaCopy** [Bali10050/Darkly#33] [General]
Would it be possible to add another branch to the repository ?
Something like 'feature-qt6'.

Currently there is only 1 main branch 'qt6' which is the release ready branch.
Having another branch would mean any active development and testing is done in that branch.
If testing goes well then promote the code up into the 'qt6' branch.
That way you can isolate dev work with production ready work ready to roll out once you tag it for release.

**Comment by Bali10050** [comment]
I created a „feature-qt6” branch, thanks for the suggestion. Also, is it okay if I tag you in the readme? I'm planning to add a section  for pre-built stuff and I plan to include a thanks to part

**Reply by DeltaCopy** [reply]
Thanks.

> Also, is it okay if I tag you in the readme?

Sure that is fine.
